<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWgt</class>
 <widget class="QWidget" name="RegisterWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(241, 255, 110, 80)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>70</number>
      </property>
      <property name="rightMargin">
       <number>70</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lab_reg">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_name">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:7px;
border-width:0.5px;
border:0.5px solid white;
</string>
        </property>
        <property name="placeholderText">
         <string>请输入您的用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_password">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:7px;
border-width:0.5px;
border:0.5px solid white;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入您的密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_repassword">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:7px;
border-width:0.5px;
border:0.5px solid white;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请再次输入您的密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_add">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:7px;
border-width:0.5px;
border:0.5px solid white;</string>
        </property>
        <property name="placeholderText">
         <string>请输入您的地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combox_render">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe 黑体 Std R</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="whatsThis">
         <string>请选择您的性别</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
  color:#1e222b;
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
  border-radius:8;
  font-size: 14px;
	font: 10pt &quot;Adobe 黑体 Std R&quot;;
}
QComboBox::drop-down {			/*下拉按钮*/
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 20px;
      border:none;
}

QComboBox QAbstractItemView{	/*列表 */
	margin-top: 8px;
	margin-left: 6px;
	margin-bottom: 6px;
	margin-right: 6px;
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:8px;
	font-size:12px;
    outline: 0px;
    color:#1e222b;
    padding-left: 16px;
    padding-top: 12px;
	padding-bottom:12px;
    
  }

QComboBox QScrollBar::vertical{ /*滑条 主体部分*/
    width:6px;
    background:rgb(255,255,255);
    border:none;
}
QComboBox QScrollBar::handle::vertical{ /*滑块主体*/
    border-radius:3px;
    background-color: rgba(30, 34, 43, 0.2);
}
QComboBox QScrollBar::handle::vertical::hover{
    background-color: rgba(30, 34, 43, 0.2);
}
QComboBox QScrollBar::add-line::vertical{/*上箭头*/
    border:none;
}
QComboBox QScrollBar::sub-line::vertical{/*下箭头*/
    border:none;
}
</string>
        </property>
        <item>
         <property name="text">
          <string>不方便透露</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVA</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_register">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 170, 127);
color:white;
border-radius:7px;
font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
